internal/core/entities/base.go

```diff
@@ -8,10 +8,10 @@ import (
 )
 
 type DBModel struct {
-	ID         uuid.UUID `gorm:"primaryKey;type:uuid" json:"id"`
+	ID         uuid.UUID `gorm:"primaryKey;type:uuid;Column:id" json:"id"`
 	CreatedAt  time.Time
 	ModifiedAt time.Time
-	DeletedAt *gorm.DeletedAt
+	DeletedAt  *gorm.DeletedAt
```

Comment:
I suggest undoing all unnecessary changes to the files that you are not the author of.